src/test/java/org/minions/devfund/bruno/html/HtmlValidatorTest.java

```diff
@@ -0,0 +1,196 @@
+package org.minions.devfund.bruno.html;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+import java.util.Queue;
+import java.util.Stack;
+
+import static org.junit.Assert.assertTrue;
+import static org.minions.devfund.bruno.html.HtmlValidator.isValidHtml;
+
+/**
+ * Tests for html validator.
+ */
+public class HtmlValidatorTest {
+
+    /**
+     * Test file 1.
+     */
+    @Test
+    public void testFile1() {
+        try {
+            Stack<HtmlTag> var1 = this.validateTestFile("test1.html");
+            if (var1 == null) {
+                Assert.fail("isValidHtml returns null for a valid HTML file input.");
+            }
+            assertTrue("isValidHtml should return an empty stack for a valid HTML file input.",
+                    var1.isEmpty());
+        } catch (Exception var2) {
+            Assert.fail("isValidHtml throws " + var2 + " for valid HTML file input");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "isValidHtml throws " 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)